File: lab3-semester1/2.cpp
// C++ Program to show swap of two no’s without using third variable

#include <iostream>

using namespace std;

int main()
{
    int x, y;
    cout<<"Enter any two numbers: "<<endl;
    
    cout<<"x = ";
    cin>>x;//takes input for x
    
    cout<<"y = ";
    cin>>y;//takes input for y
    
    //formula
    x=x*y;//first we will multiply two numbers 
    y=x/y;//this gives x*y/y=x
    x=x/y;//this gives x*y/x(look at the last equation)=system
    
    cout<<"Result: "<<endl;
    cout<<"x = "<<x<<endl;
    cout<<"y = "<<y<<endl;
    
    system("pause");
    return 0;
}

File: lab3-semester1/2.go
//Program to show swap of two integers without using third variable

package main

import "fmt"

func main() {
    var x, y int
    fmt.Println("Enter any two integers : ")
    fmt.Print("x = ")
    fmt.Scanf("%d", &x)
    fmt.Print("y = ")
    fmt.Scanf("%d", &y)
    
    x=x*y
    y=x/y
    x=x/y
    fmt.Println(" Result : ")
    fmt.Printf("x = %v\n", x)
    fmt.Printf("y = %v", y)
}
